locator/tablets.hh

```diff
@@ -158,6 +158,8 @@ struct tablet_task_info {
     db_clock::time_point request_time;
     int64_t sched_nr = 0;
     db_clock::time_point sched_time;
+    sstring repair_hosts_filter;
+    sstring repair_dcs_filter;
```

Comment:
I'm not sure this is a good idea: adding these fields here without a current use.

We can add new fields any time, no problem, but we can never remove them or change their type once added.

So even though we had a functional POC for automatic repair where these were used, I think we shouldn't add these now, instead we should wait to add them only together with the automatic repair code, once properly reviewed and agreed upon.

Reply:
Note, this is not adding columns in the table. It is adding a new field in the UDT of a system table.  We never do this before.  It is risky to add later. 

This has nothing to do with auto repair. It is not for auto repair at all. It is a basic config for tablet repair scheduler to allow user to select what hosts or dcs to repair. Currently, the tablet repair scheduler will always repair all replica nodes. 

We will almost certainly add this option support so we do not have function level regression from the user perceptive.  Currently, users can limit repair to a selected DCs or hosts with vnode tables. Without this, we could not do that with tablet tables.